liquibase-standard/src/main/java/liquibase/database/AbstractJdbcDatabase.java

```diff
@@ -1202,7 +1202,7 @@ public boolean equals(final Object o) {
 
         if (connection == null) {
             if (that.connection == null) {
```

Comment:
@arturobernalg excuse me, but I don't see the point of having this inner IF block if in any case it will return `false`.